lib/prawn/font/ttc.rb

```diff
@@ -0,0 +1,35 @@
+# font.rb : The Prawn font class
+#
+# Copyright November 2008, Jamis Buck. All Rights Reserved.
+#
+# This is free software. Please see the LICENSE and COPYING files for details.
+#
+require_relative 'ttf'
+
+module Prawn
+  class Font
+    # @private
+    class TTC < TTF
+      # Returns a list of the names of all named fonts in the given ttc file.
+      # They are returned in order of their appearance in the file.
+      #
+      def self.fonts(file)
+        TTFunk::Collection.open(file) do |f|
+          list = []
+
+          f.each do |font|
+            list << font.name.font_name.join(', ')
+          end
+
+          list
+        end
+      end
+
+      private
+
+      def read_ttf_file
```

Comment:
It may not be appropriate...but it's an inherited method from the superclass that is being overridden in the subclass. You can see that the dfont.rb implementation overrides it, too. You're right that the name is misleading, but as it was already named this before, it seems that refactoring it might be out of scope for this pull request.